image_width: 1280
image_height: 720
camera_name: zed/left
camera_matrix:
  rows: 3
  cols: 3
  data: [709.239487, 0.000000, 660.257838, 0.000000, 709.514620, 352.607432, 0.000000, 0.000000, 1.000000]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [-0.171544, 0.030474, 0.002955, -0.000407, 0.000000]
rectification_matrix:
  rows: 3
  cols: 3
  data: [0.999994, -0.002779, -0.001994, 0.002779, 0.999996, 0.000083, 0.001994, -0.000089, 0.999998]
projection_matrix:
  rows: 3
  cols: 4
  data: [696.277859, 0.000000, 676.173477, 0.000000, 0.000000, 696.277859, 363.999508, 0.000000, 0.000000, 0.000000, 1.000000, 0.000000]
